core/src/main/scala/com/nvidia/spark/rapids/tool/ClusterConfigurationStrategy.scala

```diff
@@ -26,15 +26,12 @@ import org.apache.spark.sql.rapids.tool.util.StringUtils
 case class RecommendedClusterConfig(
     numExecutors: Int,
     coresPerExec: Int,
-    numGpusPerNode: Int,
-    memoryPerNodeMb: Long // For onprem or cases where a matching CSP instance type is unavailable
+    memoryPerNodeMb: Long, // For onprem or cases where a matching CSP instance type is unavailable
+    gpuDevice: GpuDevice,
+    numGpusPerNode: Int
 ) {
-  def execsPerNode: Int = {
-    numGpusPerNode
-  }
-
   def coresPerNode: Int = {
-    coresPerExec * execsPerNode
+    coresPerExec * numGpusPerNode
```

Comment:
This calculation sounds weird. does it assume that the number of gpus represent the number of execs?